docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -91,25 +82,24 @@ You need the following to get invited to TechPass and complete the onboarding:
   4. Select **Scan a QR code**.
   5. Go back to your computer and click **Next**.
 
-  ![vendor-mfa-2](assets/support/vendor-mfa-2-new.png)
+    ![vendor-mfa-2](assets/support/vendor-mfa-2-new.png)
 
-  6. Scan the QR code on your computer screen and click **Next**. 
+  6. Scan the QR code on your computer screen and click **Next**. Your TechPass account gets activated and linked to the authenticator app.
 
-  Your TechPass account gets activated and is now linked to the authenticator app.
-    ![vendor-scan-qr-code](assets/support/vendor-mfa-3-new.png)
+     ![vendor-scan-qr-code](assets/support/vendor-mfa-3-new.png)
 
-  A number is shown on your browser.
+   A number is shown on your browser.
+   
+    ![number-mfa](assets/images/onboarding/po-non-se/number-mfa.png)
 
-   ![number-mfa](assets/images/onboarding/po-non-se/number-mfa.png)
-  
   7. On the Authenticator app, enter the number shown, and select **Yes** to authenticate your sign-in. 
```

Comment:
"Authenticator app" here starts with a cap, the "authenticator app" in line 87 is small. Maybe we should stick to one style?

select vs click? need to be consistent?

Reply:
@eu-tee  thanks! fixed. 

I have used "Select" for actions done on the mobile phone and click for actions done on the computer.